firefly/manifest.json

```
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "58481132-c79e-4659-8064-7cdaabbbc999",
+  "app_id": "firefly",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Bring your cloud Up-to-Code",
+    "title": "Firefly",
+    "media": [
+      {
+        "caption": "Full Cloud Inventory",
+        "image_url": "images/FF-inventory.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Automatic Codification",
+        "image_url": "images/FF-codification.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Detect and Fix Drifts",
+        "image_url": "images/FF-fix-drifts.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Detect and Fix Policy Violations",
+        "image_url": "images/FF-inventory.png",
+        "media_type": "image"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Autodiscovery",
+      "Category::Automation",
+      "Category::Cloud",
+      "Category::Developer Tools",
+      "Category::Google Cloud",
+      "Category::AWS",
+      "Offering::Integration"
```

You may also want to add supported OSes here, such as:

```suggestion
      "Offering::Integration",
      "Supported OS::Linux",
      "Supported OS::Mac OS",
      "Supported OS::Windows"
```